Bio/Alphabet/Reduced.py

```diff
@@ -196,8 +258,14 @@ class HPModel(Alphabet.ProteinAlphabet):
 
 
 class PC5(Alphabet.ProteinAlphabet):
+    """Reduced protein alphabet with 5 letters for physico-chemical properties.
+
+    Letters: A (Aliphatic: IVL), R (aRomatic: FYWH), C (Charged: KRDE),
+             T (Tiny: GACS), D (Diverse: TMQNP)
+    """
+
     letters = "ARCTD"
     size = 5
 
 
-hp_model = HPModel()
+pc5 = PC5()
```

Comment:
That does look like a bug, doesn't it.

Reply:
Yes, that was a bug, ```hp_model = HPModel()``` is already called after defining the ```HPModel``` class.